errors/static-dir-deprecated.md

```diff
@@ -0,0 +1,47 @@
+# Static directory is deprecated
+
+#### Why This Error Occurred
+
+In versions prior to 9.0.6 the `static` directory was used to serve static assets in a Next.js application. This has been deprecated in favor of a `public` directory.
+
+The reason we want to support a `public` directory instead is to not require the `/static` prefix for assets anymore and there is no reason to maintain both paths.
+
+#### Possible Ways to Fix It
+
+Rename your `static` directory to public and update any URLs pointing to these files to not have the `/static` prefix
```

Comment:
We could also point out that moving the `static` directory to `public` can also do the migration without having to change the url of all the static files

Reply:
IMO this is the best migration route.

Reply:
Good point 👍